docs/sources/tempo/configuration/_index.md

```diff
@@ -445,14 +448,14 @@ metrics_generator:
             [max_live_traces: <uint64>]
 
             # Whether server spans should be filtered in or not.
-            # If enabled, only parent spans or spans with the SpanKind of `server` will be retained 
+            # If enabled, only parent spans or spans with the SpanKind of `server` will be retained
             [filter_server_spans: <bool> | default = true]
 
             # Number of blocks that are allowed to be processed concurently
             [concurrent_blocks: <uint> | default = 10]
 
             # A tuning factor that controls whether the trace-level timestamp columns are used in a metrics query.
-            # If a block overlaps the time window by less than this ratio, then we skip the columns. 
+            # If a block overlaps the time window by less than this ratio, then we skip the columns.
```

Comment:
Replace "then we skip" when "the columns are skipped".

Reply:
```suggestion
            # If a block overlaps the time window by less than this ratio, then the columns are skipped.
```